### ISSUE TITLE
Image generation

### ISSUE THREAD

**Issue by Alex-451** [big-forking-team/big-forking-bot#2]
I think if we use some image generation it could help us a lot because not many people know how to do that.  It also is kinda cool haha

**Comment by hbjydev** [comment]
Yeah, but I'm not sure this should be done for help messages. Perhaps a `profile` command could use this?
